main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/jessevdk/go-flags"
+	log "github.com/sirupsen/logrus"
+	"github.com/zmap/zgrab2/zlib"
+	"os"
+)
+
+var parser = flags.NewParser(&zlib.Options[0], flags.Default)
+
+func main() {
+	if _, err := parser.Parse(); err != nil {
+		if flagsErr, ok := err.(*flags.Error); ok && flagsErr.Type == flags.ErrHelp {
+			os.Exit(0)
+		} else {
+			log.Fatal("Error: ", err.Error())
+		}
+	}
+
+	c := zlib.MakeNewController()
```

Comment:
In line with https://github.com/zmap/zgrab2/pull/1#discussion_r1288295481 and the `MakeHandler` function elsewhere, this function should just be called `MakeController` — let's be consistent with naming conventions (in this case, `MakeThing` instead of `MakeNewThing`).